pandas/tests/io/test_parquet.py

```diff
@@ -931,6 +931,18 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    @pytest.mark.parametrize(
+        "dtype",
+        ["int64", "float64", "object", "datetime64[ns, UTC]", "bool", "Int64", "UInt8"],
+    )
+    def test_read_empty_array(self, pa, dtype):
```

Comment:
Can you please add a comment pointing back to the original issue? For example, `# GH#41241`